res/vector-icons/manifest.json

```diff
@@ -37,5 +41,13 @@
 			"type": "image\/png",
 			"density": "4.0"
 		}
-	]
+	],
+    "related_applications": [{
+        "platform": "play",
+        "url": "https://play.google.com/store/apps/details?id=im.vector.alpha",
+        "id": "im.vector.alpha"
+    }, {
+        "platform": "itunes",
```

Comment:
I wish there was a way to indicate that this isn't a known platform by the w3c. (The w3c claims to only know about "play") but I understand that without the `platform` property, the application isn't processed... Oh well :grinning: